File: backoffice/FondaDomain/Domain/Currency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Runtime.Serialization;

namespace com.ds201625.fonda.Domain
{
    /// <summary>
    /// Tipo de moneda del Restaurante
    /// </summary>
	public class Currency : NounBaseEntity
    {

        /// <summary>
        /// Simbolo de la moneda de un restaurante
        /// </summary>
        private string _symbol;

		public Currency () : base () { }
        [DataMember]
        public virtual string Symbol
        {
            /// <summary>
            /// Obtiene el simbolo de la moneda usada en un restaurante
            /// </summary>
            get { return _symbol; }
            /// <summary>
            /// Asigna el simbolo de la moneda usada en un restaurante
            /// </summary>
            /// <value>Recibe simbolo de la moneda usada en un restaurante</value>
            set { _symbol = value; }
        }
			
    }
}

File: frontoffice/app/src/main/java/com/ds201625/fonda/domains/Currency.java
package com.ds201625.fonda.domains;

/**
 * Created by Katherina Molina on 10/05/2016.
 */
public class Currency extends NounBaseEntity{

    private String symbol;

    public Currency() {
        super();
    }

    public Currency(String symbol) {
        this.symbol = symbol;
    }

    public String getSymbol() {
        return symbol;
    }

    public void setSymbol(String symbol) {
        this.symbol = symbol;
    }
}
